tests/test_encode_decode.py

```diff
@@ -254,9 +256,24 @@ def test_rgb_hsv_round_trip(self, r, g, b):
         self.assertColorsValid(h=(h, h2), s=(s, s2), v=(v, v2))
 
 
+text_strategy = st.text(alphabet=st.characters(blacklist_categories=["Cc", "Cs"]))
+
+json_strategy = st.recursive(
```

Comment:
I'd call this something like `plistlib_data`, and include `datetime.datetime`s too.

Reply:
I tried to add `datetime.datetime`,  but it fails for this
```
datetime.datetime(2000, 1, 1, 0, 0) != datetime.datetime(2000, 1, 1, 0, 0, 0, 1)

Expected :datetime.datetime(2000, 1, 1, 0, 0, 0, 1)
Actual   :datetime.datetime(2000, 1, 1, 0, 0)
```

Basically, in `plistlib` a `datetime` object gets converted to string like this.
```
def _date_to_string(d):
    return '%04d-%02d-%02dT%02d:%02d:%02dZ' % (
        d.year, d.month, d.day,
        d.hour, d.minute, d.second
    )
```
Hence, the microsecond information is lost. This is causing the test case to fail.

Reply:
Presumably you could map the datetime strategy such that it's truncated to the nearest microsecond?

Reply:
Nearest minute, you mean?  I think `.map(lambda d: d.replace(seconds=0, microseconds=0)` would do it.

Reply:
Done.

Reply:
Looks like it's to the nearest second based on the code. The repr is just truncating seconds because the value is 0.